css/style.css

```diff
@@ -0,0 +1,17 @@
+#title{
+    font-family: verdana;
+    text-align: center;
+  }
+  #description{
+    text-align: center;
+    font-style: italic;
+    font-weight: bold;
+    color: orange;
+  }
+  .surv{
+    background: var(--color-darkblue-alpha);
+    padding: 2.5rem 0.625rem;
+    border-radius: 0.25rem;
+    text-align: center;
+  }
+  
```

Comment:
I think you should try and arrange your code in a more readable way but in all, you did well thanks.